readthedocs.yml

```diff
@@ -0,0 +1,11 @@
+formats:
+    - none
+build:
+    image: latest
+requirements_file: requirements.txt
+python:
+   version: 3.6
+   use_system_site_packages: true
+   setup_py_install: true
+   extra_requirements:
```

Comment:
there are no extra requirements for docs in `setup.py`, so line 10-11 can be removed, as well as line 8 (I don't think we will need system pacakges for building the docs)